mlir/lib/Dialect/Rock/Transforms/AccelEmitter.cpp

```diff
@@ -629,7 +629,7 @@ RegsAsMatrixSubTiles WmmaEmitter::computeOutputTransforms(
     mlir::rock::swapThreadIdAndIteration(
         toMatrixC, /*mBlocks=*/bidGridLengths[1], /*nBlocks=*/bidGridLengths[2],
         computeMPerThread, computeNPerThread, mPerBlock, nPerBlock,
-        doSwapThreadIterSubDimsForM, doSwapThreadIterSubDimsForN,
+        doSwapThreadIterSubDimsForM, doSwapThreadIterSubDimsForM,
```

Comment:
Why ?

Reply:
Yeah, I got confused, sorry. Let me close this PR and investigate further